src/libs/SidebarUtils.ts

```diff
@@ -451,8 +460,71 @@ function getOptionData({
 
     return result;
 }
+function getReportBeginningOfChatHistoryMessage(report: OnyxEntry<Report>): string | undefined {
```

Comment:
This function is going to be hard to maintain. Let's include its functionality in `getWelcomeMessage` making that function return both the welcome message in parts and concatenated